제가 해 본 과정이고, 코드는 당연히 사람마다 모두 다를 수 있습니다! 결과물이 같다면, 참고만 해주시면 됩니다 :)

In [ ]:
import numpy as np
import pandas as pd

In [ ]:
# 구글 코랩 사용시
from google.colab import drive
drive.mount('/content/drive')

Mounted at /content/drive


## 1. 데이터 불러오기

1) category_name.csv 파일과 KR_youtube_trending_data.csv 파일을 각각 불러오기

In [ ]:
cat = pd.read_csv("/content/drive/MyDrive/EDA/data0712/category_name.csv")
# 파일이 저장된 주소로부터 불러오기

In [ ]:
cat.head() # 자료 확인

kind                         etag  id     snippet.title  \
0  youtube#videoCategory  IfWa37JGcqZs-jZeAyFGkbeh6bc   1  Film & Animation   
1  youtube#videoCategory  5XGylIs7zkjHh5940dsT5862m1Y   2  Autos & Vehicles   
2  youtube#videoCategory  HCjFMARbBeWjpm6PDfReCOMOZGA  10             Music   
3  youtube#videoCategory  ra8H7xyAfmE2FewsDabE3TUSq10  15    Pets & Animals   
4  youtube#videoCategory  7mqChSJogdF3hSIL-88BfDE-W8M  17            Sports   

   snippet.assignable         snippet.channelId  
0                True  UCBR8-60-B28hp2BmDPdntcQ  
1                True  UCBR8-60-B28hp2BmDPdntcQ  
2                True  UCBR8-60-B28hp2BmDPdntcQ  
3                True  UCBR8-60-B28hp2BmDPdntcQ  
4                True  UCBR8-60-B28hp2BmDPdntcQ

In [ ]:
df = pd.read_csv("/content/drive/MyDrive/EDA/data0712/KR_youtube_trending_data.csv")

In [ ]:
df.head(5)

video_id                                             title  \
0  uq5LClQN3cE                                       안녕하세요 보겸입니다   
1  I-ZbZCHsHD0                                부락토스의 계획 [총몇명 프리퀄]   
2  9d7jNUjBoss                                   평생 반성하면서 살겠습니다.   
3  3pI_L3-sMVg                              안녕하세요 꽈뚜룹입니다.   
4  zrsBjYukE8s  박진영 (J.Y. Park) When We Disco (Duet with 선미) M/V   

            publishedAt                 channelId       channelTitle  \
0  2020-08-09T09:32:48Z  UCu9BCtGIEr73LXZsKmoujKw              보겸 BK   
1  2020-08-12T09:00:08Z  UCRuSxVu4iqTK5kCh90ntAgA                총몇명   
2  2020-08-10T09:54:13Z  UCMVC92EOs9yDJG5JS-CMesQ        양팡 YangPang   
3  2020-08-11T15:00:58Z  UCkQCwnkQfgSuPTTnw_Y7v7w      꽈뚜룹 Quaddurup   
4  2020-08-11T09:00:13Z  UCaO6TYtlC8U5ttz62hTrZgg  JYP Entertainment   

   categoryId         trending_date  \
0          24  2020-08-12T00:00:00Z   
1           1  2020-08-12T00:00:00Z   
2          22  2020-08-12T00:00:00Z   
3          24  2020-08-12T00:00:00Z   
4          10  2020-08-12T00:00:00Z   

                                                tags  view_count   likes  \
0                                          보겸|bokyem     5947503   53326   
1                총몇명|재밌는 만화|부락토스|루시퍼|총몇명 프리퀄|총몇명 스토리      963384   28244   
2  양팡|양팡유튜브|팡튜브|가족시트콤|양팡가족|양팡가족시트콤|양팡언니|현실남매|현실자매...     2950885   17974   
3  꽈뚜룹|한국여행기|quaddurup|뚜룹이|korea|southkorea|vlog|...     1743374   36893   
4  JYP Entertainment|JYP|J.Y.Park|JYPark|박진영|선미|S...     3433885  353337   

   dislikes  comment_count                                  thumbnail_link  \
0    105756         139946  https://i.ytimg.com/vi/uq5LClQN3cE/default.jpg   
1       494           3339  https://i.ytimg.com/vi/I-ZbZCHsHD0/default.jpg   
2     68898          50688  https://i.ytimg.com/vi/9d7jNUjBoss/default.jpg   
3      1798           8751  https://i.ytimg.com/vi/3pI_L3-sMVg/default.jpg   
4      9763          23405  https://i.ytimg.com/vi/zrsBjYukE8s/default.jpg   

   comments_disabled  ratings_disabled  \
0              False             False   
1              False             False   
2              False             False   
3              False             False   
4              False             False   

                                         description  
0                                                NaN  
1  오늘도 정말 감사드립니다!!총몇명 스튜디오 - 총몇명, 십제곱, 5G민, MOVE혁...  
2                                                NaN  
3                앞으로 좀 더 깔끔한 영상제작 약속 드리겠습니다.늘 감사드립니다  
4  MelOn http://kko.to/TWyXd7zYjSpotify https://s...

## 2. df에 대해 탐구해보아요

In [ ]:
#행과 열이 각각 몇 개인지 확인하기
nRow, nCol = df.shape
print(f'There are {nRow} rows and {nCol} columns')

There are 137554 rows and 16 columns


In [ ]:
#열 이름 16개 모두 무엇인지 출력해보기
df.columns

Index(['video_id', 'title', 'publishedAt', 'channelId', 'channelTitle',
       'categoryId', 'trending_date', 'tags', 'view_count', 'likes',
       'dislikes', 'comment_count', 'thumbnail_link', 'comments_disabled',
       'ratings_disabled', 'description'],
      dtype='object')

In [ ]:
#요약 정보 확인하기
df.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 137554 entries, 0 to 137553
Data columns (total 16 columns):
 #   Column             Non-Null Count   Dtype 
---  ------             --------------   ----- 
 0   video_id           137554 non-null  object
 1   title              137554 non-null  object
 2   publishedAt        137554 non-null  object
 3   channelId          137554 non-null  object
 4   channelTitle       137554 non-null  object
 5   categoryId         137554 non-null  int64 
 6   trending_date      137554 non-null  object
 7   tags               137554 non-null  object
 8   view_count         137554 non-null  int64 
 9   likes              137554 non-null  int64 
 10  dislikes           137554 non-null  int64 
 11  comment_count      137554 non-null  int64 
 12  thumbnail_link     137554 non-null  object
 13  comments_disabled  137554 non-null  bool  
 14  ratings_disabled   137554 non-null  bool  
 15  description        134011 non-null  object
dtypes: bool(2), int64(5)

- 결측값이 있는 열 : description (총 row가 137554인 것을 위에서 확인했으므로 알 수 있음)
- categoryId의 Dtype이 int64
- 열 ‘publishedAt’, 열 ‘trending_date’ 둘 다 object 형태

In [ ]:
# + 아래와 같은 방식으로 각 컬럼의 자료형을 확인할 수 있음
for col in df.columns:
    print('{}: {}'.format(col, df[col].dtype))

video_id: object
title: object
publishedAt: object
channelId: object
channelTitle: object
categoryId: int64
trending_date: object
tags: object
view_count: int64
likes: int64
dislikes: int64
comment_count: int64
thumbnail_link: object
comments_disabled: bool
ratings_disabled: bool
description: object


In [ ]:
# 분석과는 앞으로 관련이 없을 열‘thumbnail_link’ 제거하기
df = df.drop(["thumbnail_link"],axis=1)

In [ ]:
# 'title' 의 고유한 개수 찾기
print('고유한 title의 개수는 '+str(len(df['title'].unique()))+'개입니다.')

고유한 title의 개수는 18079개입니다.


In [ ]:
# 'video_id' 의 고유한 개수 찾기
print('고유한 video_id의 개수는 '+str(len(df['video_id'].unique()))+'개입니다.')

고유한 video_id의 개수는 16930개입니다.


In [ ]:
# 이거 더 생각해봐라 도대체 둘이 왜 다른거냐?
# + 추가 : 각 title이 몇 개씩 들어가있을까? 
df.groupby('title').count()

In [ ]:
#‘comments_disabled’열이 True라면, ‘comment_count’도 0일 것임. 이것이 사실인지 확인해보기.
print(df[df.comments_disabled==True]['comment_count'].head())
print('-'*30)

#같은 맥락으로, ‘ratings_disabled’열이 True라면, ‘likes’, ‘dislikes’ 모두 0일 것임. 이것 또한 사실인지 확인해보기.
print(df[df.ratings_disabled==True][['likes','dislikes']].sample(5))

33     0
47     0
191    0
205    0
349    0
Name: comment_count, dtype: int64
------------------------------
       likes  dislikes
85257      0         0
84665      0         0
38239      0         0
45889      0         0
73658      0         0


In [ ]:
#‘comment_count’도 0이면. ‘comments_disabled’열이 True 일 것인가?
print(df[df.comment_count==0]['comments_disabled'].value_counts())
print('-'*30)

#같은 맥락으로‘likes'가 0이라면,‘ratings_disabled’열이 True일 것인가?
print(df[df.likes==0]['ratings_disabled'].value_counts())

In [ ]:
# 열 ‘publishedAt’, 열 ‘trending_date’ 둘 다 object 형태임을 위에서 확인하였으니, 이를 모두 datatime64[ns] 형태로 변환하기.

print('전처리 이전의 데이터')
print(df[['publishedAt','trending_date']].head(5))
df['publishedAt']=pd.to_datetime(df.publishedAt)
df['publishedAt'] = df['publishedAt'].dt.tz_convert(None)
df['trending_date']=pd.to_datetime(df.trending_date)
df['trending_date'] = df['trending_date'].dt.tz_convert(None)
print()
print('전처리 이후의 데이터')
print(df[['publishedAt','trending_date']].head(5))

In [ ]:
df.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 137554 entries, 0 to 137553
Data columns (total 15 columns):
 #   Column             Non-Null Count   Dtype         
---  ------             --------------   -----         
 0   video_id           137554 non-null  object        
 1   title              137554 non-null  object        
 2   publishedAt        137554 non-null  datetime64[ns]
 3   channelId          137554 non-null  object        
 4   channelTitle       137554 non-null  object        
 5   categoryId         137554 non-null  int64         
 6   trending_date      137554 non-null  datetime64[ns]
 7   tags               137554 non-null  object        
 8   view_count         137554 non-null  int64         
 9   likes              137554 non-null  int64         
 10  dislikes           137554 non-null  int64         
 11  comment_count      137554 non-null  int64         
 12  comments_disabled  137554 non-null  bool          
 13  ratings_disabled   137554 non-null  bool    

In [ ]:
# 열 ‘categoryId’를 int type이 아닌 category type로 변환하기
df['categoryId'] = df['categoryId'].astype('category')

In [ ]:
df.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 137554 entries, 0 to 137553
Data columns (total 15 columns):
 #   Column             Non-Null Count   Dtype         
---  ------             --------------   -----         
 0   video_id           137554 non-null  object        
 1   title              137554 non-null  object        
 2   publishedAt        137554 non-null  datetime64[ns]
 3   channelId          137554 non-null  object        
 4   channelTitle       137554 non-null  object        
 5   categoryId         137554 non-null  category      
 6   trending_date      137554 non-null  datetime64[ns]
 7   tags               137554 non-null  object        
 8   view_count         137554 non-null  int64         
 9   likes              137554 non-null  int64         
 10  dislikes           137554 non-null  int64         
 11  comment_count      137554 non-null  int64         
 12  comments_disabled  137554 non-null  bool          
 13  ratings_disabled   137554 non-null  bool    

In [ ]:
df.shape

(137554, 15)

In [ ]:
df = df[df['view_count']!=0]
df.shape

(137541, 15)

In [ ]:
df = df.reset_index(drop=True)
df['tag_num'] = 0
# 태그 몇 개인지 구하기
for i in range(len(df)):
  if(df['tags'][i] == '[None]'):
    df['tag_num'][i] = 0
  else:df['tag_num'][i] = len(df['tags'][i].split('|'))

In [ ]:
#일단하지않기
#df.drop(columns=['comments_disabled','ratings_disabled'],axis=1,inplace=True)

## 3. cat, df의 categoryId와 관련한 문제 해결해보기!

In [ ]:
# 1) df에서 각 category가 몇 개씩 있는지 출력하고, 
print(df['categoryId'].value_counts())
print('-'*30)

# 2) 전체 중에서의 각 category가 차지하는 비율 출력하기
print(df['categoryId'].value_counts(normalize=True))

24    47880
22    21699
10    14787
17    10544
23     8626
25     6795
26     5972
20     5085
1      4476
15     2793
27     2691
28     2293
19     1966
2      1708
29      239
Name: categoryId, dtype: int64
------------------------------
24    0.348081
22    0.157749
10    0.107500
17    0.076654
23    0.062710
25    0.049399
26    0.043416
20    0.036967
1     0.032540
15    0.020305
27    0.019563
28    0.016670
19    0.014293
2     0.012417
29    0.001737
Name: categoryId, dtype: float64


In [ ]:
# cat에서의 열 ‘id’와 df의 열 ‘categoryId’가 같은 내용임을 확인
cat.head()

kind                         etag  id     snippet.title  \
0  youtube#videoCategory  IfWa37JGcqZs-jZeAyFGkbeh6bc   1  Film & Animation   
1  youtube#videoCategory  5XGylIs7zkjHh5940dsT5862m1Y   2  Autos & Vehicles   
2  youtube#videoCategory  HCjFMARbBeWjpm6PDfReCOMOZGA  10             Music   
3  youtube#videoCategory  ra8H7xyAfmE2FewsDabE3TUSq10  15    Pets & Animals   
4  youtube#videoCategory  7mqChSJogdF3hSIL-88BfDE-W8M  17            Sports   

   snippet.assignable         snippet.channelId  
0                True  UCBR8-60-B28hp2BmDPdntcQ  
1                True  UCBR8-60-B28hp2BmDPdntcQ  
2                True  UCBR8-60-B28hp2BmDPdntcQ  
3                True  UCBR8-60-B28hp2BmDPdntcQ  
4                True  UCBR8-60-B28hp2BmDPdntcQ

In [ ]:
# 최종 결과물은 df로 저장하며, cat에서의 ‘snippet.title’ 열이, df의 ‘category_name’열이 되도록 합치기.
df= df.merge(cat,how ='left',left_on= 'categoryId',
                                 right_on='id').rename(columns= {'snippet.title':'category_name'})
df.columns

Index(['video_id', 'title', 'publishedAt', 'channelId', 'channelTitle',
       'categoryId', 'trending_date', 'tags', 'view_count', 'likes',
       'dislikes', 'comment_count', 'comments_disabled', 'ratings_disabled',
       'description', 'kind', 'etag', 'id', 'category_name',
       'snippet.assignable', 'snippet.channelId'],
      dtype='object')

In [ ]:
# cat으로부터 합쳐진 ‘snippet.title’ (df에서의 ‘category_name’) 외의 열은 모두 없애기.
df = df.drop(['kind', 'etag', 'id','snippet.assignable', 'snippet.channelId'],axis=1)

In [ ]:
print(df.columns)
print(df.shape)

Index(['video_id', 'title', 'publishedAt', 'channelId', 'channelTitle',
       'categoryId', 'trending_date', 'tags', 'view_count', 'likes',
       'dislikes', 'comment_count', 'comments_disabled', 'ratings_disabled',
       'description', 'category_name'],
      dtype='object')
(137554, 16)


df[’category_name’]의 결측값 확인

In [ ]:
# category_name의 고유한 값들 확인하기
df['category_name'].unique()

array(['Entertainment', 'Film & Animation', 'People & Blogs', 'Music',
       'Comedy', 'Education', 'News & Politics', 'Sports', nan, 'Gaming',
       'Travel & Events', 'Pets & Animals', 'Science & Technology',
       'Howto & Style', 'Autos & Vehicles'], dtype=object)

In [ ]:
# categoryId가 29인 경우의 category_name 확인하기
df[df.categoryId==29].category_name

17        NaN
150       NaN
303       NaN
476       NaN
640       NaN
         ... 
133281    NaN
133499    NaN
133700    NaN
133938    NaN
134151    NaN
Name: category_name, Length: 239, dtype: object

In [ ]:
# + 바로 위의 사실 한 번 더 확인하기.
df.isna().sum()

video_id                0
title                   0
publishedAt             0
channelId               0
channelTitle            0
categoryId              0
trending_date           0
tags                    0
view_count              0
likes                   0
dislikes                0
comment_count           0
comments_disabled       0
ratings_disabled        0
description          3543
category_name         239
dtype: int64

In [ ]:
# 열 ‘category_name’이 결측값인 행의 title 출력해보고 내용 확인하기
df[df.category_name.isna()]['title'].unique()

array(['폭우로 인한 침수 상황시 대처 방법은? [행정안전부X사물궁이]',
       '동선 숨겼다가 고발을 당했다. 2억이 나왔다ㅣ넋 나간 가족',
       '[김광일의 입] 文 지지층, “니가 나한테 이럴 수 있어?”', '제가 전생에 무슨 죄가 있어 이 고생을 할까요?',
       '서울시청 가서 세리머니 하는 홍보맨ㅣ충주사과를 찾아라 서울편',
       '6.25전쟁 70주년 기념 유엔참전용사 감사영상 - Thanks to UN veterans.',
       '도경수 병장 30문 30답', '페트병 색깔이 투명해지고 있다고? ㅣ환경부 x 사물궁이',
       '사막마을 벼수확(5번째 이야기) / 참 까지 얻어먹고~~', '사막마을 벼 수확(6번째 이야기).  벼 탈곡기',
       '2020년 제야의 종 온라인 타종행사 | 보신각 현장 행사는 진행하지 않습니다.',
       '2020년 제야의 종 온라인 타종행사 | 2021년 새해맞이 | 보신각 현장 행사는 진행하지 않습니다.',
       '[4K][ENG Sub] BTS - With Seoul, 전지적서울시점 시즌3 하이라이트 모음',
       '[법륜스님의 즉문즉설 제 1670회] 남편과 함께 있을 때 가슴이 답답하고 머리가 멍합니다',
       '🥦팥시루떡[Adzuki Bean Rice Cake]💛밥솥을 이용해 밥하듯 쉽게해요💛칼밥상#183',
       '[Eng sub] 🥦팥시루떡[Adzuki Bean Rice Cake]💛밥솥을 이용해 밥하듯 쉽게해요💛칼밥상#183',
       '(고화질) 보너스편!! 실제상황 1인칭 경찰 시점, 응급환자 이송중인 구급차 길을 뚫어주는 싸이카 교통경찰과 시민들',
       '[법륜스님의 즉문즉설 제 1706회] 저는 한이 많이 쌓여있어요',
       '방탄소년단 - 지속가능발전목표 (SDGs) 모멘트(moment) -  | 유엔',
       'BTS Shine Spotlight on the United

In [ ]:
df['category_name'] = df['category_name'].fillna('Nonprofits & Activism')

In [ ]:
df.category_name.isna().any()

False

In [ ]:
df_unique = pd.DataFrame(columns = ['video_id', 'title', 'publishedAt', 'channelId', 'channelTitle',
       'trending_date', 'tags', 'view_count', 'likes', 'dislikes',
       'comment_count', 'description', 'category_name', 'tag_num', 'period'])
#period 열을 추가!

for i in range(len(df['video_id'].unique())):
  df_unique = df_unique.append(df[df['video_id']==df['video_id'].unique()[i]].tail(1), ignore_index=True)
  df_unique['period'][i] = len(df[df['video_id']==df['video_id'].unique()[i]])

## 4. 요약통계량 살펴보기 (이후부터 결과물은 개인 차가 존재함)

In [ ]:
df.describe()

view_count         likes       dislikes  comment_count
count  1.375540e+05  1.375540e+05  137554.000000   1.375540e+05
mean   1.618720e+06  7.406458e+04     870.091957   8.841621e+03
std    6.166372e+06  4.292023e+05    9655.556540   1.125411e+05
min    0.000000e+00  0.000000e+00       0.000000   0.000000e+00
25%    3.356400e+05  5.743000e+03       0.000000   5.470000e+02
50%    6.521940e+05  1.157800e+04     129.000000   1.215000e+03
75%    1.307116e+06  2.634875e+04     350.000000   2.870000e+03
max    2.963142e+08  1.646425e+07  879358.000000   6.939302e+06

In [ ]:
df.groupby('categoryId').median()

view_count    likes  dislikes  comment_count
categoryId                                              
1             759342.0  15420.5     165.0         1665.5
2             459026.0   4289.5     123.5          851.0
10           1081822.0  57687.0     147.0         3914.0
15            517127.0  12346.0     125.0         1056.0
17            653979.0   6561.0      99.0         1174.5
19            500747.0  10067.5      65.0         1127.0
20            508748.0   7980.0      79.0         1304.0
22            548826.0  12087.0     121.0         1062.0
23            734805.5  13231.0     166.0         1345.0
24            700801.0  11758.0     127.0         1148.0
25            811548.0   9145.0     207.0         2063.0
26            498987.5   9249.5     108.0          366.0
27            441622.0   9406.0     168.0          812.0
28            447054.0   6591.0     173.0          955.0
29            321082.0   7632.0      55.0          571.0

In [ ]:
df.groupby('categoryId').mean()

view_count          likes     dislikes  comment_count
categoryId                                                         
1           1.098933e+06   23122.434987   386.874665    2820.174486
2           7.542978e+05    9432.513466   370.252927    1503.310890
10          6.079885e+06  439184.165618  4261.412727   58516.820721
15          7.388543e+05   15344.965986   213.144647    1550.316148
17          1.083339e+06   15103.255501   336.757113    2457.487955
19          7.637694e+05   18275.155646   175.853510    1757.895727
20          1.297482e+06   38469.234415   734.631858    3433.956735
22          9.210724e+05   34847.271625   490.200055    2870.315360
23          1.158998e+06   36769.260607   795.539532    3334.783329
24          1.176115e+06   36080.916061   430.765873    3179.880138
25          1.177876e+06   14672.774834   399.787049    3772.898160
26          7.238494e+05   15489.348459   239.201440     965.924481
27          9.466088e+05   14889.859532   307.484950    1604.532887
28          1.567604e+06   39549.261230  1249.734409    2262.019625
29          8.399390e+05   69324.079498   490.682008    1466.188285

## 5. 전처리가 끝난 결과물을 csv로 저장 

In [ ]:
df.head()

video_id                                             title  \
0  uq5LClQN3cE                                       안녕하세요 보겸입니다   
1  I-ZbZCHsHD0                                부락토스의 계획 [총몇명 프리퀄]   
2  9d7jNUjBoss                                   평생 반성하면서 살겠습니다.   
3  3pI_L3-sMVg                              안녕하세요 꽈뚜룹입니다.   
4  zrsBjYukE8s  박진영 (J.Y. Park) When We Disco (Duet with 선미) M/V   

          publishedAt                 channelId       channelTitle  \
0 2020-08-09 09:32:48  UCu9BCtGIEr73LXZsKmoujKw              보겸 BK   
1 2020-08-12 09:00:08  UCRuSxVu4iqTK5kCh90ntAgA                총몇명   
2 2020-08-10 09:54:13  UCMVC92EOs9yDJG5JS-CMesQ        양팡 YangPang   
3 2020-08-11 15:00:58  UCkQCwnkQfgSuPTTnw_Y7v7w      꽈뚜룹 Quaddurup   
4 2020-08-11 09:00:13  UCaO6TYtlC8U5ttz62hTrZgg  JYP Entertainment   

   categoryId trending_date  \
0          24    2020-08-12   
1           1    2020-08-12   
2          22    2020-08-12   
3          24    2020-08-12   
4          10    2020-08-12   

                                                tags  view_count   likes  \
0                                          보겸|bokyem     5947503   53326   
1                총몇명|재밌는 만화|부락토스|루시퍼|총몇명 프리퀄|총몇명 스토리      963384   28244   
2  양팡|양팡유튜브|팡튜브|가족시트콤|양팡가족|양팡가족시트콤|양팡언니|현실남매|현실자매...     2950885   17974   
3  꽈뚜룹|한국여행기|quaddurup|뚜룹이|korea|southkorea|vlog|...     1743374   36893   
4  JYP Entertainment|JYP|J.Y.Park|JYPark|박진영|선미|S...     3433885  353337   

   dislikes  comment_count  comments_disabled  ratings_disabled  \
0    105756         139946              False             False   
1       494           3339              False             False   
2     68898          50688              False             False   
3      1798           8751              False             False   
4      9763          23405              False             False   

                                         description     category_name  
0                                                NaN     Entertainment  
1  오늘도 정말 감사드립니다!!총몇명 스튜디오 - 총몇명, 십제곱, 5G민, MOVE혁...  Film & Animation  
2                                                NaN    People & Blogs  
3                앞으로 좀 더 깔끔한 영상제작 약속 드리겠습니다.늘 감사드립니다     Entertainment  
4  MelOn http://kko.to/TWyXd7zYjSpotify https://s...             Music

In [ ]:
df.to_csv('/content/drive/MyDrive/EDA/data0712/youtube_0717.csv',index=False)

In [ ]:
df_unique.to_csv('/content/drive/MyDrive/EDA/data0712/youtube_unique_0717.csv',index=False)



---



---



수고하셨습니다.